src/main/java/com/trackit/TrackItController.java

```diff
@@ -0,0 +1,17 @@
+package com.trackit;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
```

Comment:
Added a basic controller that handles the "/" path. "index" is still required in the templates folder